src/Management/src/EndpointBase/Middleware/EndpointMiddleware.cs

```diff
@@ -8,87 +8,91 @@
 using System;
 using System.Linq;
 using System.Text.Json;
+using System.Text.Json.Serialization;
 
 namespace Steeltoe.Management.Endpoint.Middleware;
-
 public class EndpointMiddleware<TResult>
 {
     protected IEndpoint<TResult> endpoint;
     protected ILogger logger;
     protected IManagementOptions managementOptions;
 
-    public EndpointMiddleware(IManagementOptions managementOptions, ILogger logger = null)
-    {
-        this.logger = logger;
-        this.managementOptions = managementOptions ?? throw new ArgumentNullException(nameof(managementOptions));
-        if (this.managementOptions is ManagementEndpointOptions options)
+        public EndpointMiddleware(IManagementOptions managementOptions, ILogger logger = null)
```

Comment:
Was the whitespace change in this file intentional?

Reply:
No it was the result of a bad merge.